### ISSUE TITLE
User forum

### ISSUE THREAD

**Issue by RayAmini** [ru-ccmt/eDMFT#2]
Dear fellow users,

This is by far the best DMFT code I have seen for ab-initio electronic structure calculations. I suggest we help each other out running the code here on github!

**Comment by SreenivasIKST** [comment]
Hi,
Is there any proper way to run the parallel job of DMFT. 
The serial job is going fine but when I try to run in parallel the code is exiting some kind of python error.
Is there any proper job submission script for parallel job?

Thank you.

**Comment by RayAmini** [comment]
Hi!

I will start a new issue for this question.
